### ISSUE TITLE
Segmentation mask

### ISSUE THREAD

**Issue by henokyen** [zorzi-s/PolyWorldPretrainedNetwork#20]
Hello There,

Thank you for sharing your work. 

In your paper, it was indicated that corner detection is modeled as a segmentation task using weighted binary cross-entropy loss. It is also indicated that training the corner detection part needs polygonal annotations. 
My question is what are these polygonal annotations? Do they represent each corner as a single pixel with a value of 1 and leave other pixes with a value of 0? or are they just a binary polygon, meaning pixels on and inside the polygon are 1?

Thanks

**Comment by zorzi-s** [comment]
@henokyen the ground truth segmentation mask is simply a sparse array of zeros with each corner represented as a single pixel with value of 1 (see Section 4 subsection Detection, and Figure 6 in the supplementary)

**Comment by henokyen** [comment]
Thanks
